tests/test_account_chart.py

```diff
@@ -0,0 +1,99 @@
+"""
+Unit tests for account charts accessor and mutator methods.
```

Comment:
Use either 'accounts' or 'account chart' (singular) instead of 'account charts' (plural).